src/client/java/teammates/client/scripts/scalabilitytests/InstructorCourseEnrollPageScaleTest.java

```diff
@@ -0,0 +1,57 @@
+package teammates.client.scripts.scalabilitytests;
+
+import org.testng.annotations.Test;
+import teammates.common.util.AppUrl;
+import teammates.common.util.Const;
+import teammates.common.util.Logger;
+import teammates.client.scripts.util.Stopwatch;
+import teammates.test.cases.browsertests.BaseUiTestCase;
+import teammates.test.driver.FileHelper;
+import teammates.test.pageobjects.InstructorCourseEnrollPage;
+
+import java.io.IOException;
+
+/**
+ * SUT: {@link Const.ActionURIs#INSTRUCTOR_COURSE_ENROLL_PAGE}.
+ *
+ * <p>Run InstructorCourseEnrollPageScaleTestDataGenerator.java before running the tests.
+ */
+public class InstructorCourseEnrollPageScaleTest extends BaseUiTestCase {
+
+    private static final String DATA_FOLDER_PATH = "src/client/java/teammates/client/scripts/scalabilitytests/data/";
```

Comment:
Why is there a need to load data from a file? Can't we generate data on the fly?